charts/zero-notifcation-service/templates/external-secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.externalSecret.enabled -}}
+{{- $fullName := include "zero-notification-service.fullname" . -}}
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: {{ $fullName }}
+spec:
+  backendType: {{ .Values.externalSecret.backendType }}
+  {{- if .Values.externalSecret.dataFrom }}
+  dataFrom:
+  {{- range .Values.externalSecret.dataFrom }}
```

Comment:
instead of ranging through we should be able to use `{{ .Valutes.externalSecret.dataFrom | toYaml }}`

Reply:
oh nice thats much better 👍